src/org/parosproxy/paros/network/HttpRequestHeader.java

```diff
@@ -70,6 +71,13 @@
 
 public class HttpRequestHeader extends HttpHeader {
 
+    /**
+     * The {@code Accept} response header.

```

Comment:
This needs a tweak, request instead.

Reply:
I will make it a point to not push changes before I go to sleep. Will update

Reply:
Don't worry @tushar-97 we've all been there..